src/blr/cli/bamtofq.py

```diff
@@ -0,0 +1,93 @@
+"Writes paired fq files from BAM files"
```

Comment:
```suggestion
"Writes paired FASTQ files from BAM files"
```
This got changed back for some reason